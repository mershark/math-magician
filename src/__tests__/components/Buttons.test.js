import React from 'react';
import { render } from '@testing-library/react';
import Button from '../../components/Buttons';

test('Button component renders correctly', () => {
  const { container } = render(<Button onClick={() => {}}>Click Me</Button>);
  expect(container).toMatchSnapshot();
});
