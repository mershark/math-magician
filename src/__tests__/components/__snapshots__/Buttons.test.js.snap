// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component renders correctly 1`] = `
<div>
  <button
    class="calculator-button "
    type="button"
  >
    Click Me
  </button>
</div>
`;
